### PR TITLE
[feature]component for managing patient identifiers

### PR DESCRIPTION
**Pull request by Ewanjiru**
## JIRA TICKET NAME:

PDMO-5:  [Component for managing patient Identifiers](https://issues.openmrs.org/browse/PDMO-5)

## SUMMARY:

This component will consume REST api to display the existing patient identifiers and the identifier types.
It will also include functionality for adding,editing,deleting and undeleting identifiers.

### CODE REVIEW

File: app/js/components/identifiers.jsx [hunk 1]
```diff
@@ -0,0 +1,482 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ * Software distributed under the License is distributed on an "AS IS"
```
Comment (dkayiwa) [inline app/js/components/identifiers.jsx]: This license is not the one we are using

Comment (Ewanjiru) [reply app/js/components/identifiers.jsx]: Oh,, meant to change that.

Comment (Ewanjiru) [reply app/js/components/identifiers.jsx]: @dkayiwa I have corrected.

**Comment by dkayiwa** [comment]
Can you always include the ticket number in your commits?

**Comment by Ewanjiru** [comment]
@dkayiwa  The commit message has been corrected.

**Pull request merged**